Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
Turns out, I can't. Make [does not handle filenames with characters well](http://lists.gnu.org/archive/html/bug-make/2004-10/msg00010.html), so specifying the target names breaks it completely. Not specifying the target names means we end up in the same out-of-sync situation with the added downside of redoing all the work all the time.